examples/package.json

```diff
@@ -88,36 +88,36 @@
   "dependencies": {
     "@ably/chat": "^0.5.0",
     "@ably/spaces": "^0.4.0",
-    "@tailwindcss/postcss": "^4.0.14",
     "ably": "^2.5.0",
     "cors": "^2.8.5",
-    "franken-ui": "^2.0.0-internal.42",
+    "franken-ui": "^2.0.0",
+    "minifaker": "^1.34.1",
     "nanoid": "^5.0.7",
-    "next": "^15",
-    "postcss-combine-duplicated-selectors": "^10.0.3",
-    "postcss-sort-media-queries": "^5.2.0",
     "react": "^18",
     "react-dom": "^18",
     "react-icons": "^5.4.0",
-    "ts-node": "^10.9.2",
+    "react-router-dom": "^6.22.2",
```

Comment:
Brought in react router as the most general example of a routing solution for react, nothing fancy